_config.yml

```diff
@@ -11,7 +11,12 @@ markdown: kramdown
 
 # Custom Variables
 
+main:
+    plugins: "assets/plugins"
+    css: "assets/css"
+    js: "assets/js"
+
 path:
```

Comment:
Why did you put `../ ` I know what it's for but on the page itself you can call out stuff like `{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }} ` so you can have it dynamically handle this on build.